igvc/package.xml

```diff
@@ -33,6 +33,8 @@
   <!--   <run_depend>message_runtime</run_depend> -->
   <!-- Use test_depend for packages you need only for testing: -->
   <!--   <test_depend>gtest</test_depend> -->
+
+  <!-- Required-->
```

Comment:
I think this comment is redundant since everything in this file is required... that is why it is in the file

Reply:
I'll remove it.